### ISSUE TITLE
IU Xray split

### ISSUE THREAD

**Issue by thangnguyenminh** [uzh-dqbm-cmi/ARGON#1]
Hello,
Thanks for your great work on this research topic.

Looking back on the IU Xray dataset, I found out that the IU Xray dataset has not been divided into three parts, training data, validating data, and testing data at the beginning. 
Thus, to  ensure consistent comparison, I highly appreciate it if you could give me lists of subsets (train, val, and test set) which has been used to make the experiment's results in your paper.

**Comment by nooralahzadeh** [comment]
Hi,  
We used the dataset that is shared in  https://drive.google.com/file/d/1c0BXEuDy8Cmm2jfN0YYGkQxFZd2ZIoLg/view 
by https://github.com/cuhksz-nlp/R2Gen.

**Comment by thangnguyenminh** [comment]
Hi @nooralahzadeh. Thank you so much for providing this split. I am gonna close this issue now.
